pkg/router/metrics/metrics.go

```diff
@@ -99,10 +104,13 @@ func (l Listener) authorizeHandler(protected http.Handler) http.Handler {
 		scopedRecord.User = user
 		authorized, reason, err := l.Authorizer.Authorize(scopedRecord)
 		if authorized != authorizer.DecisionAllow || err != nil {
+			fmt.Printf("#### 1f\n")
 			if !ok || errors.IsUnauthorized(err) {
```

Comment:
Checking unauthorized on this error isn't correct since you get that when the router can't issue a SAR.

`ok` is the wrong `ok` post-rebase.  I'll fix that up.  Once I fix those two up, this should be ok.